api/room/schema_query.py

```diff
@@ -147,7 +147,7 @@ def resolve_room_schedule(self, info, calendar_id, days):
             events=resource[1]
         )
 
-    @Auth.user_roles('Admin')
+    @Auth.user_roles("Admin", "Default User")
```

Comment:
Since we don't need admin authentication anymore to view room analytics data, I don't think admin_location_for_analytics_view method is necessary. As a matter of fact, we would need user location for analytics view if and only if necessary.

Reply:
The `admin_location_for_analytics_view method` is used in other files. I also tried removing it but the tests still failed.

Reply:
I'm with Bonifase on this, it the method is required then the name should be changed at least so that it reflects all users' locations.